include/mbedtls/cipher.h

```diff
@@ -1,7 +1,7 @@
 /**
  * \file cipher.h
  *
- * \brief The generic cipher wrapper.
+ * \brief This file contains the generic cipher wrapper.
```

Comment:
This is pre-existing, and doesn't have to be addressed at this time, but this file isn't really a 'wrapper', in my view, but an abstraction interface.

I would propose different text:

```
 * \brief This file contains an abstraction interface for use with the cipher
 * primitives provided by the library. It provides a common interface to all of
 * the available cipher operations.
```

The existing text is fine if you want to leave it as it is.

Reply:
Your suggestion is much better. Applied.